.github/GitHub.shproj

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Copy of https://github.com/SonarSource/sonar-dotnet/blob/master/.github/GitHub.shproj